frontend/src/metabase/query_builder/components/LimitPopover.jsx

```diff
@@ -35,7 +35,7 @@ const LimitPopover = ({ limit, onChangeLimit, onClose, className }) => (
       vertical
       value={limit == null ? "maximum" : "custom"}
       options={[
-        { name: t`Show maximum`, value: "maximum" },
+        { name: t`Show maximum (first 2,000)`, value: "maximum" },
```

Comment:
@paulrosenzweig Any thoughts on where a constant like that should live in the FE so I can share it around?

Reply:
I don't have strong feelings. Either a relevant existing place like `metabase-lib/lib/Question` or a general "constants" file like you added with the collections work. I guess that general file could also live in metabase-lib since this feels more general than any component or part of the app.

I see we also have 2000 hardcoded in `QueryVisualization.jsx`.